docs/pretty.md

```diff
@@ -0,0 +1,77 @@
+# Pretty Printing
+
+By default, Pino log lines are newline delimited JSON. This is perfect for
+production usage and long term storage. It's not so great for development
+environments. Thus, Pino logs can be prettified by using a Pino prettifier
+module like [`pino-pretty`][pp]:
+
+```sh
+$ cat app.log | pino-pretty
```

Comment:
are we going with `pino-pretty`, or `pino` since it's default prettifer?

Reply:
add a line to the effect of "For almost all situations, this is the recommended way to log. The programmatic API is mostly for integration purposes of other CLI based prettifiers"